internal/client/repo_client.go

```diff
@@ -0,0 +1,46 @@
+//
```

Comment:
i think this can be under `pkg/handler/collector/github` subpath

Reply:
Well, once we support Gitlab as well as some of the other things I don't think that makes sense. These are internal wrappers around Github's stuff.